code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -115,9 +115,10 @@
 		M << "<span class='danger'>Your chameleon-projector deactivates.</span>"
 	master.disrupt()
 
-/obj/effect/dummy/chameleon/ex_act() //ok now THATS some serious protection against explosions right here
+/obj/effect/dummy/chameleon/ex_act(var/severity) //no longer bomb-proof
 	for(var/mob/M in src)
 		M << "<span class='danger'>Your chameleon-projector deactivates.</span>"
+		M.ex_act(severity)

```

Comment:
I tested it and properly took damage.

However, I did not check what happens if the user is gibbed or gets dismembered, so that spawn might be best